Formula/qcachegrind.rb

```diff
@@ -22,20 +22,32 @@ class Qcachegrind < Formula
   depends_on "graphviz"
   depends_on "qt@5"
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
   def install
-    spec = (ENV.compiler == :clang) ? "macx-clang" : "macx-g++"
-    spec << "-arm64" if Hardware::CPU.arm?
-    cd "qcachegrind" do
-      system "#{Formula["qt@5"].opt_bin}/qmake", "-spec", spec,
-                                               "-config", "release"
-      system "make"
-
-      if OS.mac?
-        prefix.install "qcachegrind.app"
-        bin.install_symlink prefix/"qcachegrind.app/Contents/MacOS/qcachegrind"
-      else
-        bin.install "qcachegrind/qcachegrind"
-      end
+    args = ["-config", "release", "-spec"]
+    os = OS.mac? ? "macx" : OS.kernel_name.downcase
+    compiler = case ENV.compiler.to_s.split("-").first
+    when :gcc
```

Comment:
```suggestion
    compiler = if ENV.compiler.to_s.start_with?("gcc")
```
This is probably simpler.

Reply:
ok